src/event-storage.js

```diff
@@ -0,0 +1,47 @@
+// @flow
+/* global window */
+
+const storageKey = 'fusion-events';
+
+type EventType = {|
+  type: mixed,
+  payload: mixed,
+|};
+
+// localStorage wrappers *exported only for testing*
```

Comment:
Can we make a class for the localStorage interface and ideally typed? btw not sure what does "exported only for testing" mean when we actually need them in the browser plugin

Reply:
The exports at the bottom are public and the ones at the top are pseudo private...

I like the interface idea. Will explore further tomorrow!